### ISSUE TITLE
5 extensions sync, but not the 6th

### ISSUE THREAD

**Issue by bdelaney** [shanalikhan/code-settings-sync#377]
**Visual Studio Code Version :** [ 1.14.2 ]
**Code Settings Sync Version :** [ 2.8.2 ]
**Operating System :** [ Win10 64bit v1703]
**Occurs On:** [Download ]
**Proxy Enabled:** [ No ]
**GIst Id:** [ 47877511aa729fdfe42e0562e5827bfa ]

Just added Settings Sync for first time yesterday. Wanted to sync 3 copies of VS Code. All PCs are running same version of Windows, all are same version of VS Code.
In my GIST the 6 extensions listed are:
_EXTENSIONS ADDED :

Angular2 - Version :2.4.0
code-settings-sync - Version :2.8.2
typescript-hero - Version :1.3.0
vscode-icons - Version :7.12.0
vscode-wakatime - Version :1.1.12
wallaby-vscode - Version :1.0.56_

When I download using Settings Sync on second/third PC, the typescript-hero extension is NOT loaded. The rest of the extensions are.

In my GIST, I see the extension enumerated:
`  {

    "metadata": {

      "id": "38583aca-2ba1-417f-80fc-68ca8d717fb4",

      "publisherId": "c5d551f6-3d2b-41b9-8b4b-6224c4fcfb87",

      "publisherDisplayName": "Christoph Bühler"

    },

    "name": "typescript-hero",

    "publisher": "rbbit",

    "version": "1.3.0"

  },`

**Comment by shanalikhan** [comment]
Is there any exception logged in developer console ?

**Comment by bdelaney** [comment]
No, there was no exception. I'm going to try this again with same 3 PCs after I add some other extensions.
For troubleshooting, first I will remove all extensions from one PC and try to duplicate it with just the single failing extension.

**Comment by bdelaney** [comment]
Tried with just the single extension. Seemed to work, install, although two errors showed.
![syncsettings errors](https://user-images.githubusercontent.com/595730/29366701-5bd767f8-8260-11e7-9b4b-58052056bb92.PNG)
May try and reproduce later today with a full set of extensions.

**Comment by shanalikhan** [comment]
Always close the settings and all other files that will be affected from sync else code wiill merge the older version and new version. close affected files and try sync again.

**Comment by atomgao** [comment]
I have the same problem, and there was no exception.
vscode 1.15.1 x32 or x64
windows10 [10.0.15063]

**Comment by shanalikhan** [comment]
@atomgao whats your gist ID, paste here so i can reproduct

**Comment by atomgao** [comment]
@shanalikhan 
gist: d4c422181e8037b1141c42ab98df766a
I am using it in company and home. 
home: total(53), disabled(1)
company:toal(50), disabled(1)

**Comment by atomgao** [comment]
It's worked right in v2.8.3
